### PR TITLE
env variables added

### PR DESCRIPTION
**Pull request by faisalamin001**
env variables added for Nav component

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
-}
+  env: {
```
Comment (juanigallo) [inline next.config.js:5]: Hi! According to next documentation (https://nextjs.org/docs/basic-features/environment-variables) the recommended way to add env variables is in the `.env.local` file

File: components/Nav/index.js [hunk 2]
```diff
@@ -5,10 +5,11 @@ export default function Nav() {
   return (
     <nav className={styles.nav}>
       <div className={styles.innerNav}>
-        <Link href="/">
+        <Link href='/'>
```
Comment (juanigallo) [inline components/Nav/index.js:8]: Please change your linter settings to avoid unexpected changes

File: pages/index.js [hunk 3]
```diff
@@ -11,8 +11,8 @@ export default function Home({ entries }) {
     <>
       <Head>
         <title>Juani Gallo - Blog</title>
-        <meta name="description" content="Juani Gallo's blog" />
-        <link rel="icon" href="/favicon.ico" />
+        <meta name='description' content="Juani Gallo's blog" />
```
Comment (juanigallo) [inline pages/index.js]: Please change your linter settings to avoid unexpected changes

File: pages/index.js [hunk 4]
```diff
@@ -11,8 +11,8 @@ export default function Home({ entries }) {
...
+        <link rel='icon' href='/favicon.ico' />
```
Comment (juanigallo) [inline pages/index.js:15]: Please change your linter settings to avoid unexpected changes

**Review by juanigallo — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (faisalamin001) [reply next.config.js:5]: I have added .env.local file but you know its a private file so git do not push this file, Create .env.local file and paste `APP_LOGO=JuaniGallo
APP_NAME=NotionBlog` in it. and env variables must be configured in Next.js which I have already configured.
Screenshot 2022-09-22 at 8 13 28 AM

File: pages/index.js (hunk 4 above)
Comment (faisalamin001) [reply pages/index.js:15]: Changed

File: next.config.js (hunk 1 above)
Comment (juanigallo) [reply next.config.js:5]: Awesome, then let's remove the key env from this file. As it's not needed after Next 9.4

Comment (faisalamin001) [reply next.config.js:5]: **`.env.local`** provides values to backend
**`next.config.js`** provides values to frontend
If I delete it then the app wouldn't be able to show it in UI.

File: components/Nav/index.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-import Link from "next/link";
-import styles from "./style.module.scss";
+import Link from 'next/link';
```
Comment (juanigallo) [inline components/Nav/index.js:1]: Please fix the linting in here

File: components/Nav/index.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
+import styles from './style.module.scss';
```
Comment (juanigallo) [inline components/Nav/index.js:2]: Please fix the linting in here

File: pages/index.js [hunk 7]
```diff
@@ -1,8 +1,8 @@
-import Head from "next/head";
-import styles from "../styles/Home.module.scss";
-import { Client } from "@notionhq/client";
-import Entry from "../components/Entry";
-import Nav from "../components/Nav";
+import Head from 'next/head';
```
Comment (juanigallo) [inline pages/index.js:1]: Please fix the linting in all the imports of this file

File: pages/index.js [hunk 8]
```diff
@@ -11,13 +11,13 @@ export default function Home({ entries }) {
...
         <title>Juani Gallo - Blog</title>
-        <meta name='description' content="Juani Gallo's blog" />
+        <meta name='description' content='Juani Gallos blog' />
         <link rel='icon' href='/favicon.ico' />
       </Head>
       <Nav />
       <div className={styles.container}>
         <section className={styles.entries}>
-          {entries.length == 0 && "There are no entries"}
+          {entries.length == 0 && 'There are no entries'}
```
Comment (juanigallo) [inline pages/index.js:20]: Please fix the linting in here

Comment (faisalamin001) [reply pages/index.js:20]: It's already in single quote, should change it to double quote?

File: next.config.js (hunk 1 above)
Comment (juanigallo) [reply next.config.js:5]: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

**Pull request closed without merging**